developers/blobstreamx-offchain.md

```diff
@@ -1,18 +1,18 @@
 ---
-description: Learn how to integrate your L2's offchain logic with Blobstream
+description: Learn how to integrate your L2's offchain logic with Blobstream X
 ---
 
-# Integrate with Blobstream client
+# Integrate with Blobstream X client
 
-## Blobstream demo rollup
+## BlobstreamX demo rollup
 
-Rollups can use the Blobstream for DA by posting their data to Celestia and then
+Rollups can use the Blobstream X for DA by posting their data to Celestia and then
```

Comment:
nit: "Rollups can use Blobstream X for DA by .."

Reply:
https://github.com/celestiaorg/docs/pull/1379/commits/d7d2f14e872ee4fcc5a06079a3e5b0c29840e0f2 ✅